website/source/docs/agent/dns.html.md

```diff
@@ -57,8 +57,9 @@ we can instead use `foo.node.consul.` This convention allows for terse
 syntax where appropriate while supporting queries of nodes in remote
 datacenters as necessary.
 
-For a node lookup, the only records returned are A records containing
-the IP address of the node.
+For a node lookup, the only records returned are A and AAAA records 
```

Comment:
I added language for 'AAAA' records as well since it is in available in the code. Feel free to remove that in case it is not officially supported yet.